<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="mocap" exec="mocap_node" name="optitrack" output="screen">
    <!-- General Parameters -->
    <param name="tf_parent_frame" value="world" />
    <param name="mocap_wrt_parent_frame" value="0 0 0 0.0 0 0.0" />
    <param name="body_wrt_mocap_body" value="0 0 0 0.0 0.0 0" />
    <param name="num_markers_missing_tol" value="4" />
    <param name="q_diff_threshold" value="0.1" />
    <param name="skipped_meas_pos_tol" value="3" />
    <param name="skipped_meas_att_tol" value="20" />
    <param name="pub_status" value="true" />
    <param name="pub_twist" value="true" />
    <param name="pub_accel" value="true" />
    <param name="tf_broadcast" value="true" />

    <!-- OptiTrack-specific parameters -->
    <param name="client" value="optitrack" />
    <param name="local" value="192.168.1.119" /> <!-- IP address of Sikorsky NIC connected to OptiTrack -->
    <param name="server" value="192.168.1.245" /> <!-- OptiTrack windows computer -->
    <param name="multicast_group" value="239.255.42.99" /> <!-- Determined by OptiTrack settings -->
    <param name="command_port" value="1510" /> <!-- Determined by OptiTrack settings -->
    <param name="data_port" value="1511" /> <!-- Determined by OptiTrack settings -->
  </node>

</launch>
